extensions/amp-subscriptions/0.1/amp-subscriptions.css

```diff
@@ -61,22 +61,14 @@ html.i-amphtml-subs-grant-unk [subscriptions-action] {
   display: none !important;
 }
 
-/** Action: login. Only show when logged out. */
-html:not(.i-amphtml-subs-loggedin-no) [subscriptions-action="login"] {
+[subscriptions-action] {
   display: none !important;
 }
 
-/** Action: logout. Only show when logged in. */
-html:not(.i-amphtml-subs-loggedin-yes) [subscriptions-action="logout"] {
-  display: none !important;
-}
-
-/** Action: subscribe. Only show when not a subscriber. */
-html:not(.i-amphtml-subs-subscriber-no) [subscriptions-action="subscribe"] {
-  display: none !important;
+[i-amphtml-subs-display][subscriptions-action] {
+  display: block !important;
```

Comment:
The action should be able to use any "display" value it'd like. We need to express it in reverse. I.e. display:none unless i-amphtml-subs-display is specified.

Also, we need the same for `subscriptions-actions` and such?